pom.xml

```diff
@@ -538,11 +538,24 @@
             <version>1.9.13</version>
         </dependency>
 
-        <!-- hadoop compilation dependency for CodecPool -->
+        <!-- hadoop compilation dependencies for CodecPool and AvroGenericRecordReader -->
         <dependency>
             <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-core</artifactId>
-            <version>0.20.2</version>
+            <artifactId>hadoop-common</artifactId>
+            <version>3.1.0</version>
```

Comment:
Why not the latest 3.1 branch which is 3.1.4?

Reply:
I looked at dependency which was used by hive 3.1.2. I can make it 3.1.4. As it is just for compilation it does not matter at all I guess.